### ISSUE TITLE
Disable is_external and nofollow in the URL Control

### ISSUE THREAD

**Issue by mathetos** [elementor/elementor#11214]
## Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
I'm building custom widgets for Elementor for GiveWP. We have the need to have two widgets:
* Register form
* Login form

We have existing shortcodes that I'm using for these Elementor widgets. One attribute of the shortcode that I need a widget control for is the redirect url. This need to just be the raw URL. I've got it all working great so far. 

The problem is that the Elementor URL Control supports `is_external` and `nofollow` and they seemingly can't be disabled at all. In this specific use-case (and I imagine plenty others) this will just create confusion for the user. 

Obviously I could just hide the gear button with CSS, but that feels hacky. I'd rather be able to have a control that has URL validation, the Page/Post Autocomplete, but NO additional controls. 

Perhaps [the docs](https://code.elementor.com/classes/elementor-control_url/) aren't updated correctly, or my code isn't quite right, but it seems that the `show_external` arg should dictate that, but it's just not working as intended. 

```
$this->add_control(
			'login_url',
			[
				'label' => __('Redirect URL', 'dw4elementor'),
				'type' => \Elementor\Controls_Manager::URL,
				'show_external' => false,
				'default' => [
					'url' => ''
				],
				'description' => __('The url users will be redirected to after they login.', 'dw4elementor'),
			]
		);
```
**Describe the solution you'd like**
I'd like to be able to programmatically disable the external "Link Options" controls in the "URL Control". Perhaps that is done with the `show_external` and this is rather a bug than a feature request. 

**Describe alternatives you've considered**
I've considered creating a custom control, but that feels over-engineered when I really want to simplify an existing control.

I might hide the control with CSS in the interim to avoid user confusion.

**Additional context**
This is the control I'd like to have disabled completely. Using the code above, that "Link Options" still appears, which seems either like an oversight, or a bug. 
![image](https://user-images.githubusercontent.com/930724/80176092-5aa73c80-85ac-11ea-9149-a1ed66b23618.png)

**Comment by rami-elementor** [comment]
Hi @mathetos ,

This can be done using a simple `TEXT` control and an `input_type` argument:

```
$this->add_control(
	'login_url',
	[
		'label' => esc_html__( 'Redirect URL', 'dw4elementor' ),
		'type' => \Elementor\Controls_Manager::TEXT,
		'input_type' => 'url',
		'placeholder' => esc_html__( 'https://example.com/', 'dw4elementor' ),
		'description' => __('The url users will be redirected to after they login.', 'dw4elementor'),
	]
);
```

**Comment by drinkmaker** [comment]
To disable `is_external` and `nofollow` for Elementor URL Control pass `'options' => false`
Example:
```php
/** Custom URL. */
$this->add_control(
	'custom_url',
	[
		'label'             => esc_html__( 'Custom URL', 'text-domain' ),
		'label_block'       => true,
		'type'              => Controls_Manager::URL,
		'options'           => false,
		'dynamic'           => [ 'active' => true ],
		'placeholder'       => esc_html__( 'Enter URL', 'text-domain' ),
		'default'           => '#',
	]
);
```

**Comment by soyket** [comment]
@rami-elementor, it seems this doesn't work anymore. 
https://developers.elementor.com/docs/editor-controls/control-url/

You can see the error in your video widget as well. When you choose the option Self Hosted > Then choose external > The video URL field will have attribute + other controls but the code base says options => False. So this is no longer working.

**Comment by rami-elementor** [comment]
@soyket

The video widget has the following code:

```php
		$this->add_control(
			'external_url',
			[
				'label' => esc_html__( 'URL', 'elementor' ),
				'type' => Controls_Manager::URL,
				'autocomplete' => false,
				'options' => false,
				'label_block' => true,
				'show_label' => false,
				'dynamic' => [
					'active' => true,
					'categories' => [
						TagsModule::POST_META_CATEGORY,
						TagsModule::URL_CATEGORY,
					],
				],
				'placeholder' => esc_html__( 'Enter your URL', 'elementor' ),
				'condition' => [
					'video_type' => 'hosted',
					'insert_url' => 'yes',
				],
			]
		);
```

it disables the `options` and the `autocomplete` to allow the user to specify an external URL.

The rendered HTML is not an `<a>` link rather a `<video>` element, therefore the `options` are not required.

**Comment by soyket** [comment]
@rami-elementor May I know if the options => false or other array for the options array work for you? It doesn't at my end. 

As for the video widget, what I was saying is the video widget should not have the attribute + link open> option in new tab field, but it has them - as the options array is set to false - https://github.com/elementor/elementor/blob/af3b3fb14b01febca2a13f48718267f9fe521d41/includes/widgets/video.php#L234C1-L235C1

https://www.loom.com/share/90d6c274fcae4374a0364be4f0556b18

The expected behavior is - Open in new window, Add nofollow, Custom Attributes -> should not be there. 

Let me know if that helps in understanding what I am reporting.

**Comment by drinkmaker** [comment]
I confirm, `'options' => false`, doesn't work anymore for URL Control.
I checked it on the latest version of the Elementor 3.15.2

**Comment by nicholaszein** [comment]
Hi @drinkmaker and @soyket!

We have great news! 🙌 

### 📢 We're happy to announce that the regression in the functionality you've reported was resolved in Elementor v3.17! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
